std/experimental/ndslice/selection.d

```diff
@@ -98,6 +98,23 @@ template pack(K...)
 ///
 @safe @nogc pure nothrow unittest
 {
+    import std.experimental.ndslice : sliced, Slice, pack;
+    import std.range: iota;
+
+    auto r = (3 * 4 * 5 * 6).iota;
+    auto a = r.sliced(3, 4, 5, 6);
+    auto b = a.pack!(2);
```

Comment:
you indented `auto b` with 5 spaces + usually the style is `a.pack!2` when only one template parameter is given.